pkg/server/server.go

```diff
@@ -101,6 +102,8 @@ func (s *SSIServer) instantiateRouter(service svcframework.Service) error {
 		return s.ManifestAPI(service)
 	case svcframework.Presentation:
 		return s.PresentationAPI(service)
+	case svcframework.Submission:
+		return s.SubmissionAPI(service)
```

Comment:
I had a working assumption that the `PresentationAPI` was going to be including the `SubmissionAPI` similar to how the `ApplicationAPI` is under the `ManifestAPI`. They should be the same service because they are a part of the same spec/group of functionality IMO